src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -45,6 +45,69 @@ def setUpClass(cls):
         cls.ws_id = int(config.get("app_tests", "public_ws_id"))
         cls.app_input_ref = config.get("app_tests", "test_input_ref")
         cls.batch_app_id = config.get("app_tests", "batch_app_id")
+        cls.bulk_run_good_inputs = [
+            {
+                "app_id": "kb_uploadmethods/import_fastq_sra_as_reads_from_staging",
+                "tag": "release",
+                "version": "1.0.46",
+                "params": [
+                    {
+                        "fastq_fwd_staging_file_name": "",
+                        "fastq_rev_staging_file_name": "",
+                        "sra_staging_file_name": "",
+                        "name": "asdf",
+                        "import_type": "FASTQ/FASTA",
+                        "insert_size_mean": 0,
+                        "insert_size_std_dev": 0,
+                        "interleaved": 1,
+                        "read_orientation_outward": "0",
+                        "sequencing_tech": "Illumina",
+                        "single_genome": 1,
+                    }
+                ],
+            },
+            {
+                "app_id": "kb_uploadmethods/import_sra_as_reads_from_staging",
+                "tag": "release",
+                "version": "1.0.46",
+                "params": [
+                    {
+                        "sra_staging_file_name": "",
+                        "name": "asdf",
+                        "insert_size_mean": 1,
+                        "insert_size_std_dev": 1,
+                        "read_orientation_outward": "0",
+                        "sequencing_tech": "Illumina",
+                        "single_genome": "1",
+                    }
+                ],
+            },
+        ]
+
+    def setUp(self):
+        os.environ["KB_WORKSPACE_ID"] = self.public_ws
```

Comment:
A number of tests were just setting up the same workspace id. Consolidated here.